app/components/preNoted/suddenAlert.scss

```diff
@@ -5,6 +5,7 @@
   right: 0;
   width: 371px;
   background-color: $dark-blue0;
+  z-index: 999;
```

Comment:
is it okay with paper show's elements? (before / after scroll)

Reply:
Yes, I checked it and it worked without problems.
refCitedTab stick -> ok
ref / cited paper layout -> ok
pdf viewer -> ok